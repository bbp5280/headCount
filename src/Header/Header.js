import React from 'react';
import styles from '../index.css';
import icon from '../../assets/headcount-icon.png';

const Header = () => {
  return (
    <nav className="header">
      <img src={icon} />
      <h1>HeadCount</h1>
    </nav>
  );
};

export default Header;
