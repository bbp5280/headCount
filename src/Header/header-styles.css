@import url(https://fonts.googleapis.com/css?family=Lato);
.icon {
  background-image: url(../../assets/headcount-icon.svg); }

.header {
  background-image: linear-gradient(to right, #e64f40, #EF6544);
  padding: 25px;
  font-size: 130%;
  display: flex;
  align-items: center; }

h1 {
  color: #fff;
  font-family: 'Lato', sans-serif;
  display: inline-block; }

img {
  height: 45px;
  width: auto;
  display: inline-block;
  margin-right: 1%; }
