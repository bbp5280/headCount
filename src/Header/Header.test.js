import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import config from '../../test/setup.js';

describe('HEADER', () => {
  let wrapper = shallow(<Header />);

  it('should have a header container', () => {
    let header = wrapper.find('.header');

    expect(header.length).toEqual(1);
  });

  it('should match the Header snapshot', () => {

    expect(wrapper).toMatchSnapshot();
  });

});
