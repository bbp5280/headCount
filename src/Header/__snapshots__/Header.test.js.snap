exports[`HEADER should match the Header snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          src="headcount-icon.png" />,
        <h1>
          HeadCount
</h1>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "src": "headcount-icon.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "HeadCount",
        },
        "ref": null,
        "rendered": "HeadCount",
        "type": "h1",
      },
    ],
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            src="headcount-icon.png" />,
          <h1>
            HeadCount
</h1>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "src": "headcount-icon.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "HeadCount",
          },
          "ref": null,
          "rendered": "HeadCount",
          "type": "h1",
        },
      ],
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
